lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -4,17 +4,50 @@
 // in this document:
 // --------------------
 // +notes
+// +local variables/utilities
 // +toggling notes
+// +individual note (in context)
 // +creating/editing notes
 // +listing notes
+// +necessary, but ugly overrides
 
 // +notes:
 // --------------------
 // this Sass partial contains all of the styling needed for the in-line student notes UI.
 
-// +toggling notes
+// +local variables/utilities:
 // --------------------
+$notes-annotator-background-light: rgb(251, 251, 251); // taken from annotatorJS base colors
+$notes-annotator-background-med: rgb(214, 214, 214); // taken from annotatorJS base colors
+$notes-annotator-background-dark: rgba(122,122,122,0.6); // taken from annotatorJS base colors
+
+%notes-reset-background {
```

Comment:
Needed extends that reset (aggressively) a lot of AnnotatorJS's default vendor styling.